cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/module/DefaultModuleTaskEntry.java

```diff
@@ -16,36 +16,96 @@
 
 package de.dytanic.cloudnet.driver.module;
 
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
 import java.lang.reflect.Method;
+import org.jetbrains.annotations.NotNull;
 
-public final class DefaultModuleTaskEntry implements IModuleTaskEntry {
+public class DefaultModuleTaskEntry implements IModuleTaskEntry {
 
-  private final IModuleWrapper moduleWrapper;
+  /**
+   * A simple format which allows unique method identifiers. Format: {@code <declaring class>@<method name>()}
+   */
+  protected static final String METHOD_SIGNATURE_FORMAT = "%s@%s()";
 
-  private final ModuleTask taskInfo;
+  protected final Method jlrMethod;
```

Comment:
What does jlr mean

Reply:
java.lang.reflect

Reply:
I think something like reflectMethod is more appropriate